docs/03-tutorials/00-serverless/svls-02-create-git-function.md

```diff
@@ -133,9 +133,9 @@ Follows these steps:
 
     - Open your Namespace view. In the left navigation panel, go to **Configuration** > **Secrets** and select the **Create Secret** button.
 
-    - In the **Metadata** tab, enter the Secret name under **Name**.
+    - Open the **Advanced** view and enter the secret name and type.
```

Comment:
```suggestion
    - Open the **Advanced** view and enter the Secret name and type.
```

In the other instances, Secret is capitalized, for example [here](https://github.com/kyma-project/kyma/pull/12174/files#diff-48c000138a07f94412f9f8d07d5eda1152ea8f8c0c4cf7ebcbeceb457d1bb9e7R134).